pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -10,15 +10,15 @@
 , stdenvNoCC
 , runtimeShell
 , bintools ? null, libc ? null, coreutils ? null, gnugrep ? null
-, netbsd ? null, netbsdCross ? null
+, netbsd ? null
```

Comment:
Not to be overzealous, but  that excess of `? null` is dangerous.

Reply:
Sure, but it's been this way for sake of bootstrapping for years. I hope after I find time to redo the GCC bootstrapping to dramatically simplify all this stuff :).